src/parser/matches/arg_matches.rs

```diff
@@ -418,50 +556,6 @@ impl ArgMatches {
         !self.args.is_empty()
     }
 
-    /// Get an [`Iterator`] over groups of values of a specific option.
-    ///
-    /// specifically grouped by the occurrences of the options.
-    ///
-    /// Each group is a `Vec<&str>` containing the arguments passed to a single occurrence
-    /// of the option.
-    ///
-    /// If the option doesn't support multiple occurrences, or there was only a single occurrence,
-    /// the iterator will only contain a single item.
-    ///
-    /// Returns `None` if the option wasn't present.
-    ///
-    /// # Panics
-    ///
-    /// If the value is invalid UTF-8.
-    ///
-    /// If `id` is not a valid argument or group id.
-    ///
-    /// # Examples
-    /// ```rust
-    /// # use clap::{Command,Arg, ArgAction};
-    /// let m = Command::new("myprog")
-    ///     .arg(Arg::new("exec")
-    ///         .short('x')
-    ///         .num_args(1..)
-    ///         .action(ArgAction::Append)
-    ///         .value_terminator(";"))
-    ///     .get_matches_from(vec![
-    ///         "myprog", "-x", "echo", "hi", ";", "-x", "echo", "bye"]);
-    /// let vals: Vec<Vec<&str>> = m.grouped_values_of("exec").unwrap().collect();
-    /// assert_eq!(vals, [["echo", "hi"], ["echo", "bye"]]);
-    /// ```
-    /// [`Iterator`]: std::iter::Iterator
-    #[cfg(feature = "unstable-grouped")]
-    #[cfg_attr(debug_assertions, track_caller)]
-    pub fn grouped_values_of(&self, id: &str) -> Option<GroupedValues> {
```

Comment:
While this is an unstable feature, should we at least put a deprecatiion on this for a release or two?

It gives people an opportunity to experiment with both more easily

Reply:
I added it back, and marked it as deprecated.